frontend/src/test/scala/bloop/tasks/RunTasksSpec.scala

```diff
@@ -0,0 +1,127 @@
+package bloop.tasks
+
+import org.junit.Test
+import org.junit.Assert.assertEquals
+
+import bloop.cli.Commands
+import bloop.engine.{Interpreter, Run}
+import bloop.engine.tasks.RunTasks
+import bloop.logging.{BloopLogger, ProcessLogger, RecordingLogger}
+import bloop.tasks.ProjectHelpers.{checkAfterCleanCompilation, getProject}
+
+class RunTasksSpec {
```

Comment:
Are you not testing forked execution here?

Reply:
Duh, good catch.